app.py

```diff
@@ -449,12 +474,14 @@ def the_blast_promo_form():
         installment_period="yearly",
         key=app.config["STRIPE_KEYS"]["publishable_key"],
         bundles=bundles,
+        css=css,
     )
 
 
 @app.route("/submit-blast-promo", methods=["POST"])
 def submit_blast_promo():
     bundles = get_bundles("old")
+    css = get_css("old")
```

Comment:
Should this be get_styles? Just comparing to the other similar calls, and noticed this is the only place get_css is used.

Reply:
yes, omg good catch! I'm working on `get_bundles` doing all the work though. So I'll remove in a future commit.